modules/SofaGeneralAnimationLoop/MechanicalMatrixMapper.cpp

```diff
@@ -0,0 +1,77 @@
+/******************************************************************************
+*       SOFA, Simulation Open-Framework Architecture, version 1.0 RC 1        *
+*                (c) 2006-2011 MGH, INRIA, USTL, UJF, CNRS                    *
```

Comment:
2006-2018 ;-)

Reply:
Done with https://github.com/sofa-framework/sofa/pull/721/commits/ac71b0ea494cec0a84deeb537e0566f4e2da2919, thanks for the review!